### PR TITLE
Vcr

### PR DESCRIPTION
**Pull request by perryr16**
Mocking all API calls with VCR linked to Webmock

### CODE REVIEW

File: spec/features/admin/edit_tutorial_spec.rb [hunk 1]
```diff
@@ -4,7 +4,8 @@
   let(:tutorial) { create(:tutorial) }
   let(:admin)    { create(:admin) }
 
-  scenario "by adding a video", :js do
+  # scenario "by adding a video", :js, :vcr do
+  scenario "by adding a video",  :vcr do
```
Comment (perryr16) [inline spec/features/admin/edit_tutorial_spec.rb:8]: I am unable to get `:vcr` to work as a flag if there is already a `:js` flag. Removing the `:js` had no effect on testing, but there has to be a reason it was there.

File: Gemfile [hunk 2]
```diff
@@ -53,4 +53,9 @@ group :development do
   gem 'bundler'
 end
 
+group :test do 
+  gem 'webmock'
+  gem 'vcr'
+end
```
Comment (stellakunzang) [inline Gemfile:59]: Hooray!

File: spec/features/admin/edit_tutorial_spec.rb (hunk 1 above)
Comment (stellakunzang) [reply spec/features/admin/edit_tutorial_spec.rb:8]: I'm glad you left it there as a comment. Maybe will come into play during the javascript lesson this morning

**Review by stellakunzang — APPROVED** [review]
Yay! Thank you for doing this! Looks really simple but also really tedious.

**Pull request merged**